process-automation/latest/develop/index.md

```diff
@@ -93,6 +88,530 @@ Once the extended Digital Workspace has been fully customized and tested it can
+
+@Component({
+  selector: 'custom-template-name-column',
+  template: `
+    <mat-list>
+        <mat-list-item>
+            <div mat-line>{{ displayValue.name }}</div>
+            <div mat-line [ngStyle]="{ 'padding-top': '5px' }">
+                <span [ngStyle]="{ 'font-weight': 'bold' }">Assignee :</span>
+                <span> {{ displayValue.assignee }}</span>
+            </div>
+        </mat-list-item>
+    </mat-list>
+    `,
+  host: { class: 'adf-datatable-content-cell adf-name-column' },
+})
+export class TaskNameComponent implements OnInit {
+
+  @Input()
+  context: any;
+
+  displayValue: TaskDetailsCloudModel;
+
+  constructor() {}
+
+  ngOnInit() {
+    this.displayValue = this.context?.row?.obj;
+  }
+}
+```
+
+```typescript
+import { Component, Input, OnInit } from '@angular/core';
+import { TaskDetailsCloudModel } from '@alfresco/adf-process-services-cloud';
+
+@Component({
+  selector: 'custom-template-priority-column',
+  template: `
+    <mat-form-field>
+      <mat-label>Change Priority</mat-label>
+      <mat-select [(ngModel)]="selectedValue" name="priority">
+        <mat-option *ngFor="let priority of priorities" [value]="priority">
+          {{priority}}
+        </mat-option>
+      </mat-select>
+    </mat-form-field>
+    `,
+  host: { class: 'adf-datatable-content-cell adf-name-column' },
+})
+export class TaskPriorityComponent implements OnInit {
+
+  @Input()
+  context: any;
+
+  displayValue: TaskDetailsCloudModel;
+
+  priorities: string[] = ['None', 'High', 'Low', 'Normal'];
+  selectedValue: string;
+
+  constructor() {}
+
+  ngOnInit() {
+    this.displayValue = this.context?.row?.obj;
+    this.selectedValue = this.displayValue.priority;
+  }
+}
+```
+
+```typescript
+import { Component, Input, OnInit } from '@angular/core';
+import { ProcessInstanceCloud } from '@alfresco/adf-process-services-cloud';
+
+@Component({
+  selector: 'custom-template-duedate-column',
+  template: `
+    <mat-form-field>
+        <input matInput [matDatepicker]="picker" placeholder="Change due date" />
+        <mat-datepicker-toggle matSuffix [for]="picker"></mat-datepicker-toggle>
+        <mat-datepicker #picker></mat-datepicker>
+    </mat-form-field>
+    `,
+  host: { class: 'adf-datatable-content-cell adf-name-column' },
+})
+export class TaskDueDateComponent implements OnInit {
+
+  @Input()
+  context: any;
+
+  displayValue: TaskDetailsCloudModel;
+
+  constructor() {}
+
+  ngOnInit() {
+    this.displayValue = this.context?.row?.obj;
+  }
+}
+```
+
+Register the custom components in the [task-list.module.ts](https://github.com/Alfresco/alfresco-apps/blob/9a7c4106fb5befc05bb45e2703bd70c6a7436fb1/libs/content-ee/process-services-cloud-extension/src/lib/features/task-list/task-list.module.ts){:target="_blank"} file. For more on how to register a custom component see [Registration](https://github.com/Alfresco/alfresco-content-app/blob/develop/docs/extending/registration.md){:target="_blank"}.
```

Comment:
The first link points to a specific commit - should it link to the default branch instead?

Reply:
I confirmed that with Ankur yesterday he seems ok with where those links are going. Cheers.